### PR TITLE
river-status: add layout_name and layout_name_clear events

### PR DESCRIPTION
**Pull request by Leon-Plickat**
We've had the layout name forever and never actually send it forward to status clients.

### CODE REVIEW

**Comment by ifreund** [comment]
Is there a client that uses this?

**Comment by MaxVerevkin** [comment]
> Is there a client that uses this?

If this PR gets merged, I'll add this to `i3bar-river`

**Comment by ifreund** [comment]
> If this PR gets merged, I'll add this to `i3bar-river`

Thanks for the comment. I was mostly asking in hopes of a test client to be able to review this PR.

**Comment by MaxVerevkin** [comment]
> I was mostly asking in hopes of a test client to be able to review this PR.

If I have time I can implement it before merging.

**Comment by MaxVerevkin** [comment]
Works as expected for me. Tested with https://github.com/MaxVerevkin/i3bar-river/commit/f6727acdd4cc112c51d3dd71df61689738e31de9.

File: protocol/river-status-unstable-v1.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
     </request>
   </interface>
 
-  <interface name="zriver_output_status_v1" version="2">
+  <interface name="zriver_output_status_v1" version="3">
```
Comment (ifreund) [inline protocol/river-status-unstable-v1.xml:50]: It's actually necessary to bump this and `zriver_status_manager_v1` to version 4 for backwards compatibility. I'll fix this minor issue myself though during the rebase and merge.

**Review by ifreund — APPROVED** [review]
Thanks for the patch @Leon-Plickat and for the test client implementation @MaxVerevkin!

**Comment by MaxVerevkin** [comment]
@ifreund I think you forgot to change `since="3"` to `since="4"`.

**Comment by ifreund** [comment]
> @ifreund I think you forgot to change `since="3"` to `since="4"`.

whoops, indeed. Will push a fix to master in a minute.

**Pull request merged**